bugwarrior/services/jira.py

```diff
@@ -159,7 +159,7 @@ def get_due(self):
             return self.parse_date(self.record['fields']['duedate'])
         # Otherwise, if the issue is in a sprint, use the end date of that sprint.
         sprints = self.__get_sprints()
-        for sprint in filter(lambda e: e.get('state') != 'CLOSED', sprints):
+        for sprint in filter(lambda e: e.get('state', '').lower() != 'closed', sprints):
```

Comment:
Can we make a constant of `closed`?

Reply:
The string is generated by Jira and exported through it's API. python-jira has nothing about it and this is the only place it is used in the bugwarrior jira.py. Why would you want to make it a constant?